.github/workflows/release.yml

```diff
@@ -14,6 +14,6 @@ jobs:
     with:
       publishToBinaries: true
       mavenCentralSync: true
-      slackChannel: team-sonarqube-build
+      slackChannel: ops-analysis-experience
```

Comment:
Shouldn't we rename it to `ops-analysis-processing` at some point?

Reply:
Yes probably, and then we will have to update all the workflows from our repositories.
I'll do it later.